### DISCUSSION TITLE
如何回复一条消息

### DISCUSSION THREAD

**Discussion by chrisyy2003** [nonebot/discussions#276] [问答] [answered]
请问一下，如何实现下面的响应，没看到对应的接口
![image](https://user-images.githubusercontent.com/63803385/208076941-16905786-6ff0-4528-a1bc-9cf58768da84.png)

**Comment by yanyongyu** [comment, marked as answer]
`MessageSegment.reply` 消息id

**Reply by chrisyy2003** [reply]
没在这个类下找到这个方法或变量呢？
```
@dataclass
class MessageSegment(abc.ABC, Generic[TM]):
    """消息段基类"""

    type: str
    """消息段类型"""
    data: Dict[str, Any] = field(default_factory=dict)
    """消息段数据"""
```

**Reply by yanyongyu** [reply]
`from nonebot.adapters.onebot.v11 import MessageSegment`
